### PR TITLE
chore: refine print statement

### PR DESCRIPTION
**Pull request by steven-mi**
## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed

## Added to documentation?

- [ ] 👍 README.md
- [ ] 👍 CHANGELOG.md
- [ ] 👍 Additional documentation in /docs
- [ ] 👍 Relevant code documentation
- [ ] 🙅 no, because they aren’t needed

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog db-rocket
 
+## Version 1.3.1
+
+- Adding Markdown documentation to package description
```
Comment (gyg-martin-jewell) [inline CHANGELOG.md:9]: would this be the changelog and version for this particular PR?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Changelog db-rocket
 
+## Version 1.3.1
```
Comment (gyg-martin-jewell) [inline CHANGELOG.md:7]: doesn't match the version in setup.py

File: setup.py [hunk 3]
```diff
@@ -1,15 +1,22 @@
 import setuptools
 
+# load the README file and use it as the long_description for PyPI
+with open("README.md", encoding="utf8") as f:
+    readme = f.read()
+
 setuptools.setup(
     name="databricks-rocket",
-    version="1.3.0",
+    version="1.3.2",
```
Comment (gyg-martin-jewell) [inline setup.py:9]: we could pull this from the changelog too, the same as above

File: setup.py [hunk 4]
```diff
@@ -1,15 +1,22 @@
 import setuptools
 
+# load the README file and use it as the long_description for PyPI
+with open("README.md", encoding="utf8") as f:
+    readme = f.read()
```
Comment (gyg-martin-jewell) [inline setup.py:5]: this duplicates [this PR](https://github.com/getyourguide/db-rocket/pull/45/files), should it be here?

**Review by gyg-martin-jewell — APPROVED** [review]

**Review by meghanagyg — APPROVED** [review]

**Pull request merged**